src/GrassPokemon.java

```diff
@@ -0,0 +1,127 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class GrassPokemon extends Pokemon {
+    String name;
+    int level;
+    int hp;
+    String food;
+    String sound;
+    private static final String type = "grass";
```

Comment:
Heel goed dat je deze final en static maakt. Het is natuurlijk immutabel (final) en elke instantie heeft hier dezelfde waarde (static). 
In feite is het nu geen "variabele", maar een "constante" en volgens conventie schrijven we dat dan met hoofdletters, dus: `private static final String TYPE = "grass";`